src/components/FAQ/QAItem/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const QAItem = ({ question, answer }) => {
+  return (
+    <ul className="dp-accordion">
+      <li>
+        <span className="dp-accordion-thumb">
+          <b>{question}</b>
+        </span>
+        <div className="dp-accordion-panel">
+          <div className="dp-accordion-content">{answer}</div>
+        </div>
+      </li>
+    </ul>
+  );
+};
+
+QAItem.propTypes = {
+  question: PropTypes.object.isRequired,
+  answer: PropTypes.object.isRequired,
+};
+
+export default QAItem;
```

Comment:
We try to avoid export default, could you add this as a named export?

Reply:
:+1: